README.md

```diff
@@ -32,7 +32,9 @@ usage: duct [-h] [--version] [-p OUTPUT_PREFIX]
             [-t {all,system-summary,processes-samples}]
             command [command_args ...] ...
 
-Gathers metrics on a command and all its child processes.
+duct is a lightweight wrapper that collects execution data for an arbitrary
+command. Execution data includes execution time, system information, and
+resource usage statistics of the command and all its child processes.
```

Comment:
should mention that it samples that process and children periodically, so that information could be used to later analyze/plot command's resources consumption.  And that it might be useful to declare needed resources for allocation on HPC tasks, or for troubleshooting.  Also note that it stores all the standard output and error into external files.
Do we have somewhere listing of env vars to configure (may be in README for now)? should refer to that description here too IMHO.

Reply:
> should mention that it samples that process and children periodically, so that information could be used to later analyze/plot command's resources consumption. And that it might be useful to declare needed resources for allocation on HPC tasks, or for troubleshooting. Also note that it stores all the standard output and error into external files.

+1

> Do we have somewhere listing of env vars to configure (may be in README for now)? should refer to that description here too IMHO.

Its not listed anywhere atm, +1 to add here.